flow-typed/npm/lodash_v4.x.x.js

```diff
@@ -197,7 +197,8 @@ declare module 'lodash' {
     includes(str: string, value: string, fromIndex?: number): bool;
     invokeMap<T>(array: ?Array<T>, path: ((value: T) => Array<string>|string)|Array<string>|string, ...args?: Array<any>): Array<any>;
     invokeMap<T: Object>(object: T, path: ((value: any) => Array<string>|string)|Array<string>|string, ...args?: Array<any>): Array<any>;
-    keyBy<T, V>(array: ?Array<T>, iteratee?: ValueOnlyIteratee<T>): {[key: V]: T};
+    keyBy<T, V>(array: ?Array<T>, iteratee?: (value: T) => V): {[key: V]: T};
+    keyBy<T: Object, V: string>(array: ?Array<T>, iteratee?: V): {[key: $ElementType<T, V>]: T};
```

Comment:
Did you have a chance to submit a PR at the flow-typed repository? :)

Reply:
Not yet, but I'd like to in the next days.